src/Source/DoctrineORMQuerySourceIterator.php

```diff
@@ -20,6 +20,17 @@
 
 class DoctrineORMQuerySourceIterator implements SourceIteratorInterface
 {
+    const DATE_PARTS = [
+            'y' => 'Y',
+            'm' => 'M',
+            'd' => 'D',
+        ];
+    const TIME_PARTS = [
+            'h' => 'H',
+            'i' => 'M',
+            's' => 'S',
+        ];
```

Comment:
There is an indention issue here… weird that flintCI doesn't complain…

Reply:
cc @Soullivaneuh 

but it isn't spotted by our lint-job either 🤔  Maybe we need another config option for PHP-CS-Fixer ?